src/frontend/qt_sdl/CustomDnsQuery.h

```diff
@@ -0,0 +1,394 @@
+// DNS Query Program on Linux
```

Comment:
this code seems to come from somewhere else with no attribution

Reply:
yes, you're right. i got it from https://www.binarytides.com/dns-query-code-in-c-with-linux-sockets/

i'm sure there are better, shorter and prettier ways to implement this. i just wanted to test things out, without adding more dependencies. Than i got hyped because i got it working. I'm still kind of a noob. sry, for that